### PR TITLE
Replace lastSeen date from search with client date

### PR DESCRIPTION
**Pull request by kenoir**
In order to fix a problem with weighted search where some images are not
displayed.

This should hopefully also simplify the update polling logic.

### CODE REVIEW

**Comment by ShaunYearStrong** [comment]
👍

File: kahuna/public/js/search/results.js [hunk 1]
```diff
@@ -157,11 +157,7 @@ results.controller('SearchResultsCtrl', [
             // results), unless we're reloading a previous search in
             // which case we reuse the previous time too
 
-            // FIXME: the resolution of uploadTime is seconds, which could
-            // not be enough to avoid multiple images sharing an
-            // uploadTime and issues with duplicate results in the
-            // queried set
-            const latestTime = images.data[0] && images.data[0].data.uploadTime;
+            const latestTime = moment().toISOString();
```
Comment (akash1810) [inline kahuna/public/js/search/results.js]: too lazy to check... has `moment` been imported further up?

**Comment by akash1810** [comment]
👍

File: kahuna/public/js/search/results.js (hunk 1 above)
Comment (kenoir) [inline kahuna/public/js/search/results.js]: oh yeah

**Pull request merged**